dev/devicelab/bin/tasks/simple_animation_perf_ios__e2e_summary.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter_devicelab/tasks/perf_tests.dart';
+import 'package:flutter_devicelab/framework/adb.dart';
+import 'package:flutter_devicelab/framework/framework.dart';
+
+// TODO(CareF): add it to manifest.yaml
```

Comment:
The "simple_animation_perf" test is mainly for measuring CPU/GPU usages. Currently, those metrics only exist in the Timeline. They're not in `FrameTiming` or any similar class that's exposed directly through APIs. It seems that we have to migrate this to a hybrid e2e test?

Reply:
Maybe not necessarily. timeline is available after flutter/plugins#2947 but we will need to wait until Dan update the e2e in the framwork to `integration_test`